src/controllers/Controller.java

```diff
@@ -54,107 +51,102 @@ public void initialize(URL location, ResourceBundle resources) {
+
     }
 
-    private void initializeCategoryView(HashMap<String, CategoryModel> categoryModelHashMap) {
+    private void initializeCategoryPane(HashMap<String, CategoryModel> categoryModelHashMap) {
 
-        categoryListView.setOnKeyPressed(new EventHandler<KeyEvent>() {
+        categoryPane.setOnMouseClicked(new EventHandler<MouseEvent>() {
             @Override
-            public void handle(KeyEvent enterKeyPressed) {
-                if (enterKeyPressed.getCode().equals(KeyCode.ENTER)) {
-                    populateCategoriesList(categoryModelHashMap);
-                }
-            }
-        });
-    }
-
-    private void populateCategoriesList(HashMap<String, CategoryModel> categoriesModelHashMap) {
+            public void handle(MouseEvent event) {
 
-        ObservableList<String> observableList = FXCollections.observableArrayList();
-        for (CategoryModel categoryModel : categoriesModelHashMap.values()) {
 
-            observableList.add(categoryModel.getName());
-        }
-        categoryListView.setItems(observableList);
-    }
+                for (CategoryModel categoryModel : categoryModelHashMap.values()) {
 
-    private void initializeFoodView() {
+                    final Button tempButton = new Button(categoryModel.getName());
+                    tempButton.setMinHeight(69);
+                    tempButton.setMinWidth(150);
 
-        categoryListView.setOnMouseClicked(new EventHandler<MouseEvent>() {
-            @Override
-            public void handle(MouseEvent mouseEvent) {
-                String categoryName = categoryListView.getSelectionModel().getSelectedItem().toString();
+                    categoryPane.getChildren().addAll(tempButton);
+                    tempButton.setOnAction(new EventHandler<ActionEvent>() {
+                        @Override
+                        public void handle(ActionEvent event) {
+                            String selectedCategory = tempButton.getText();
 
-                populateFoodItemsListView(categoryName);
+                            foodItemPane.getChildren().clear();
+                            populateItemsInterface(selectedCategory, categoryModelHashMap);
+                        }
+                    });
+                }
             }
         });
     }
 
-    private void populateFoodItemsListView(String categoryName) {
+    private void populateItemsInterface(String selectedCategory, HashMap<String, CategoryModel> categoryModelHashMap) {
 
-        ObservableList<String> observableList = FXCollections.observableArrayList();
-        CategoryModel categoryModel = categoryModelHashMap.get(categoryName);
-        ArrayList<FoodItemModel> foodItemsList = categoryModel.getFoodItemsList();
+        FoodItemsService foodItemsService = new FoodItemsService();
+        ArrayList<FoodItemModel> foodItemModelArrayList = foodItemsService.get(selectedCategory);
+
+        for (FoodItemModel foodItemModel : foodItemModelArrayList) {
```

Comment:
This looks really good, however while looking at this i see lots of code duplication between the way you set up categories and the way you set up buttons.  In a larger project i would look for ways to reduce that duplication so that the code is more concise and maintainable.  But looks good.